pkg/pipeline/transform/kubernetes/kubernetes.go

```diff
@@ -291,23 +294,13 @@ func (k *KubeData) initServiceInformer(informerFactory informers.SharedInformerF
 	return nil
 }
 
-func (k *KubeData) initReplicaSetInformer(informerFactory informers.SharedInformerFactory) error {
-	k.replicaSets = informerFactory.Apps().V1().ReplicaSets().Informer()
-	// To save space, instead of storing a complete *appvs1.Replicaset instance, the
-	// informer's cache will store a *metav1.ObjectMeta with the minimal required fields
-	if err := k.replicaSets.SetTransform(func(i interface{}) (interface{}, error) {
```

Comment:
I think the SetTransform should be kept, because it reduces memory usage even more, by keeping only Name/Namespace/OwnerRefs instead of the full metadata